app/AppKernel.php

```diff
@@ -23,6 +23,7 @@ public function registerBundles()
             new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
Why do we simply can't use `dev` environment to populate the fixtures instead of allowing it in `prod`?

P.S. Sorry, maybe I don't fully understand Heroku.